build/build-windows-k8s.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+set -eo pipefail
+
+NSSM_VERSION=2.24
+NSSM_URL=https://k8stestinfrabinaries.blob.core.windows.net/nssm-mirror/nssm-${NSSM_VERSION}.zip
+
+exec_with_retry() {
+
+      local retry=0
+      local max_retries=$1
+      local cmd=${@:2}
+      local exit_code=0
+
+      while [[ ${retry} -lt ${max_retries} ]]; do
+          eval $cmd || exit_code=$?
+          if [[ ${exit_code} -eq 0 ]]; then
+              return ${exit_code}
+          fi
+          let retry=retry+1
+     done
+     if [[ ! ${exit_code} -eq 0 ]]; then
+          echo "Failed to execute command: $cmd with exit_code: ${exit_code}"
+          return ${exit_code}
+    fi
+}
+
+create_dist_dir() {
+	mkdir -p ${DIST_DIR}
+}
+
+build_kubelet() {
+	echo "building kubelet.exe..."
+	$KUBEPATH/build/run.sh make WHAT=cmd/kubelet KUBE_BUILD_PLATFORMS=windows/amd64 KUBE_VERBOSE=0
+	cp ${GOPATH}/src/k8s.io/kubernetes/_output/dockerized/bin/windows/amd64/kubelet.exe ${DIST_DIR}
+}
+
+build_kubeproxy() {
+	echo "building kube-proxy.exe..."
+	$KUBEPATH/build/run.sh make WHAT=cmd/kube-proxy KUBE_BUILD_PLATFORMS=windows/amd64 KUBE_VERBOSE=0
+	cp ${GOPATH}/src/k8s.io/kubernetes/_output/dockerized/bin/windows/amd64/kube-proxy.exe ${DIST_DIR}
+}
+
+build_kubectl() {
+	echo "building kubectl.exe..."
+    $KUBEPATH/build/run.sh make WHAT=cmd/kubectl KUBE_BUILD_PLATFORMS=windows/amd64 KUBE_VERBOSE=0
```

Comment:
indentation is off here for this line and the next

Reply:
done